### PR TITLE
Commands and Datafixer

### PR DESCRIPTION
**Pull request by OroArmor**

### CODE REVIEW

File: mappings/net/minecraft/datafixer/DataFixerMode.mapping [hunk 1]
```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/unmapped/C_qzkadpgu net/minecraft/datafixer/DataFixerMode
```
Comment (triphora) [inline mappings/net/minecraft/datafixer/DataFixerMode.mapping:1]: Is it Datafixer or DataFixer?

File: mappings/net/minecraft/datafixer/schema/Schema3087.mapping [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+		ARG 1 schema
+		ARG 2 description
+		ARG 3 target
+		ARG 4 op
+	METHOD m_dtqptazs (Lcom/mojang/datafixers/util/Pair;)Lcom/mojang/datafixers/util/Pair;
+		ARG 1 pair
+	METHOD m_flfsemwg (Lcom/mojang/serialization/Dynamic;)Lcom/mojang/serialization/Dynamic;
+		ARG 1 __dynamic
+	METHOD m_ofvuoqlm (Lcom/mojang/serialization/Dynamic;)Lcom/mojang/serialization/Dynamic;
+		ARG 1 ___dynamic
```
Comment (triphora) [inline mappings/net/minecraft/datafixer/schema/Schema3087.mapping:15]: Why do these need 2 and 3 underscores?

**Review by triphora — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/datafixer/DataFixerMode.mapping (hunk 1 above)
Comment (OroArmor) [reply mappings/net/minecraft/datafixer/DataFixerMode.mapping:1]: DataFixer

File: mappings/net/minecraft/datafixer/schema/Schema3087.mapping (hunk 2 above)
Comment (OroArmor) [reply mappings/net/minecraft/datafixer/schema/Schema3087.mapping:15]: These are deep lambda parameters in a schema, don't really worry about them

File: mappings/net/minecraft/SharedConstants.mapping [hunk 3]
```diff
@@ -35,6 +35,7 @@ CLASS net/minecraft/unmapped/C_otltijcb net/minecraft/SharedConstants
 	FIELD f_iavlnkwa ONLY_GENERATE_HALF_OF_THE_WORLD Z
 	FIELD f_inoiujxk GRAVITY_BLOCK_DUPLICATION_FIX Z
 	FIELD f_iodtkwtq ENABLE_AQUIFERS_DEBUGGER Z
+	FIELD f_isyhbtke DATAFIXER_MODE Lnet/minecraft/unmapped/C_qzkadpgu;
```
Comment (triphora) [inline mappings/net/minecraft/SharedConstants.mapping]:
```suggestion
	FIELD f_isyhbtke DATA_FIXER_MODE Lnet/minecraft/unmapped/C_qzkadpgu;
```

**Review by triphora — APPROVED** [review]

**Review by NoahvdAa — APPROVED** [review]

**Review by NoComment1105 — APPROVED** [review]

File: mappings/net/minecraft/datafixer/schema/Schema3087.mapping [hunk 4]
```diff
@@ -0,0 +1,24 @@
+CLASS net/minecraft/unmapped/C_ktmwrpbm net/minecraft/datafixer/schema/Schema3087
+	FIELD f_bepreebv target Ljava/lang/String;
+	FIELD f_rbbdwmto description Ljava/lang/String;
+	FIELD f_vdniceoq op Ljava/util/function/UnaryOperator;
```
Comment (IotaBread) [inline mappings/net/minecraft/datafixer/schema/Schema3087.mapping]: `operator` is better imo

File: mappings/net/minecraft/server/command/WardenSpawnTrackerCommand.mapping [hunk 5]
```diff
@@ -0,0 +1,16 @@
+CLASS net/minecraft/unmapped/C_nraktviw net/minecraft/server/command/WardenSpawnTrackerCommand
+	METHOD m_cnwtursr register (Lcom/mojang/brigadier/CommandDispatcher;)V
+		ARG 0 commandDispatcher
```
Comment (IotaBread) [inline mappings/net/minecraft/server/command/WardenSpawnTrackerCommand.mapping]:
```suggestion
		ARG 0 dispatcher
```

File: mappings/net/minecraft/server/command/WardenSpawnTrackerCommand.mapping [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+	METHOD m_fhzvhekf (Lcom/mojang/brigadier/context/CommandContext;)I
+		ARG 0 commandContext
```
Comment (IotaBread) [inline mappings/net/minecraft/server/command/WardenSpawnTrackerCommand.mapping]:
```suggestion
		ARG 0 context
```

File: mappings/net/minecraft/server/command/WardenSpawnTrackerCommand.mapping [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+		ARG 0 player
+	METHOD m_lrlilrtg setWarningLevel (Lnet/minecraft/unmapped/C_pennblrk;Ljava/util/Collection;I)I
+		ARG 0 source
+		ARG 1 players
+		ARG 2 warningLevel
+	METHOD m_pkdsohqh clearWarningLevel (Lnet/minecraft/unmapped/C_pennblrk;Ljava/util/Collection;)I
+		ARG 0 source
+		ARG 1 players
+	METHOD m_wrzdmfyf (Lcom/mojang/brigadier/context/CommandContext;)I
+		ARG 0 commandContext
```
Comment (IotaBread) [inline mappings/net/minecraft/server/command/WardenSpawnTrackerCommand.mapping]:
```suggestion
		ARG 0 context
```

File: mappings/net/minecraft/datafixer/schema/Schema3087.mapping (hunk 4 above)
Comment (OroArmor) [reply mappings/net/minecraft/datafixer/schema/Schema3087.mapping]:
```suggestion
	FIELD f_vdniceoq operator Ljava/util/function/UnaryOperator;
```

**Pull request merged**